src/connectRoutes.js

```diff
@@ -317,6 +329,9 @@ export default (
     }
 
     if (typeof window !== 'undefined') {
+      // `kind` is typed as ?string which will not throw if passed to .test.
+      // Typing kind as string threw other flow errrs in the codebase
+      // $FlowIssue
```

Comment:
fyi, later i'll check if we even need the `?`. it should always be there. either way, if it's not, we wont past that test anyway.